tests/PhpWordTests/Reader/MsDocTest.php

```diff
@@ -60,6 +60,22 @@ public function testLoad(): void
         self::assertInstanceOf('PhpOffice\\PhpWord\\PhpWord', $phpWord);
     }
 
+    public function testLoadHalfPointFont(): void
+    {
+        $filename = __DIR__ . '/../_files/documents/word.2526.doc';
```

Comment:
@oleibman Could you use reader.doc ?

Reply:
I can't use reader.doc because it does not contain any half-point fonts, so it doesn't demonstrate the problem. I could change it, but that might cause problems if anyone depends on it as-is. I agree that's pretty unlikely, so, if that's what you want me to do, I will go ahead and do that.

Reply:
I understand. Could you rename it as reader.font-halfpoint.doc ?

Reply:
Done, and I have also added 1.3.0.md.